api/main.py

```diff
@@ -109,14 +109,14 @@ def workflow_state(dataset_id):
     decision_task_id = str(task["id"])
     task_details = task["content"]
     task_progress = engine_decision["progress"]
+    skipped_task_to_remove = engine_decision.get("removeSkipActions")
```

Comment:
Tiny cosmetic thought, should this be: skipped_tasks_to_remove as you correctly name it in logic.py?